src/librustc_trans/adt.rs

```diff
@@ -49,11 +49,10 @@ use std::rc::Rc;
 
 use llvm::{ValueRef, True, IntEQ, IntNE};
 use rustc::ty::subst::Substs;
+use rustc::ty::layout;
 use rustc::ty::{self, Ty, TyCtxt};
-use syntax::ast;
 use syntax::attr;
 use syntax::attr::IntType;
-use abi::FAT_PTR_ADDR;
 use build::*;
 use common::*;
 use debuginfo::DebugLoc;
```

Comment:
I haven't completed this PR yet. @eddyb thought I should make it anyway so we would get diff comments and the like.  Making `Repr` not a thing is the goal, but I haven't finished and lots of places still insist on using it.